website/content/docs/nia/cli/task.mdx

```diff
@@ -0,0 +1,76 @@
+---
+layout: docs
+page_title: Task Subcommands
+sidebar_title: task
+description: >-
+  Consul-Terraform-Sync supports task subcommands for users to modify tasks while the daemon is running
+---
+# task
+
+## task enable
+
+`task enable` subcommand enables an existing task so that it will run and update task resources. If the task is already enabled, it will return a success. The subcommand generates and outputs an inspect plan of how resources will be modified if task becomes enabled. If resources changes are detected, the subcommand will ask for user approval before enabling the task. If no resources are detected, the subcommand will go ahead and enable the task.
```

Comment:
```suggestion
`task enable` subcommand enables an existing task so that it will run and update task resources. If the task is already enabled, it will return a success. The subcommand generates and outputs a Terraform plan of how resources will be modified if task becomes enabled. If resources changes are detected, the subcommand will ask for user approval before enabling the task. If no resources are detected, the subcommand will go ahead and enable the task.
```

A suggestion you don't have to accept.

Reply:
Yes! 'inspect plan' isn't quite right. I like 'a Terraform plan'. I wanted to (awkwardly) fit in that this is similar to the inspect-mode. I'll just add an additional phrase mentioning the inspect mode.